### DISCUSSION TITLE
`supabase stop` should stop all containers

### DISCUSSION THREAD

**Discussion by nahtnam** [supabase/supabase#19072] [Feature Requests]
I work on several different projects on my local computer and use `supabase start` to start them. The problem is they consistently interfere with each other and so I have to stop one before starting another. However you **must** run `supabase stop` in the directory that you started them, which is pretty annoying. It would be nice if we could have a `supabase stop --all` which would stop all supabase projects running on the computer.

That way I can just set `direnv` to run `supabase stop --all && supabase start` whenever I CD into a specific directory

**Comment by EskelCz** [comment]
`supabase stop --all` sounds good +1
